test/remote/gateways/remote_paystation_test.rb

```diff
@@ -4,7 +4,8 @@ class RemotePaystationTest < Test::Unit::TestCase
   
 
   def setup
-    @gateway = PaystationGateway.new(fixtures(:paystation))
+    @gateway      = PaystationGateway.new(fixtures(:paystation))
+    @hmac_gateway = PaystationGateway.new(fixtures(:hmac_paystation))
```

Comment:
Heya Justin - the `paystation` fixture doesn't exist either - these require production credentials, so they'd usually be added to `~/.active_merchant/fixtures.yml` if I've figured that out correctly?

I could add both to `fixtures.yml` to serve as examples?